ngraph/core/include/ngraph/opsets/opset4_tbl.hpp

```diff
@@ -158,6 +157,7 @@ NGRAPH_OP(Atanh, ngraph::op::v3)
 NGRAPH_OP(CTCLoss, ngraph::op::v4)
 NGRAPH_OP(HSwish, ngraph::op::v4)
 NGRAPH_OP(NonMaxSuppression, ngraph::op::v4)
+NGRAPH_OP(Interpolate, ngraph::op::v4)
```

Comment:
Can you move this line up so that the file is sorted alphabetically?

Reply:
Fixed.